cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +90,46 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		deny.PluginName,                       // AlwaysDeny
+		admit.PluginName,                      // AlwaysAdmit
+		autoprovision.PluginName,              // NamespaceAutoProvision
+		lifecycle.PluginName,                  // NamespaceLifecycle
+		initialization.PluginName,             // Initializers
```

Comment:
Move just after mutating.  This plugin owns the list.

Reply:
Is the recommend order wrong? `{lifecycle.PluginName, initialization.PluginName, mutatingwebhook.PluginName, validatingwebhook.PluginName}`

Reply:
`Initializers` is before `MutatingAdmissionWebhook`